Cargo.lock

```diff
@@ -345,6 +345,7 @@ dependencies = [
  "kube-client",
  "kube-runtime",
  "log",
+ "mio",
```

Comment:
It's a transitive dependency, how come it got added here? Did `cargo` do that?

Reply:
```console
$ cargo update -p mio
    Updating crates.io index
    Updating mio v0.8.10 -> v0.8.11
```

Here's the resulting diff:

```diff
diff --git a/Cargo.lock b/Cargo.lock
index 8f9c852..79c2b7a 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -1032,9 +1032,9 @@ dependencies = [
 
 [[package]]
 name = "mio"
-version = "0.8.10"
+version = "0.8.11"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "8f3d0b296e374a4e6f3c7b0a1f5a51d748a0d34c85e7dc48fc3fa9a87657fe09"
+checksum = "a4a650543ca06a924e8b371db273b2756685faae30f8487da1b56505a8f78b0c"
 dependencies = [
  "libc",
  "wasi",
```